code/modules/paperwork/paperplane.dm

```diff
@@ -32,15 +32,6 @@
 		internalPaper = new(src)
 	update_icon()
 
-/obj/item/paperplane/handle_atom_del(atom/A)
-	if(A == internalPaper)
-		var/obj/item/paper/P = internalPaper
-		internalPaper = null
-		P.moveToNullspace() //So we're not deleting it twice when deleting our contents.
-		if(!QDELETED(src))
-			qdel(src)
-	return ..()
```

Comment:
why?

Reply:
Deleted movables are moved to nullspace and `loc.Exited()` is executed, which is why we don't actually need `handle_atom_del()` doing what `Exited()` already does.